src/Invoice_GenUI/Invoice_GenUI/Models/Validation/UsernameValidation.cs

```diff
@@ -0,0 +1,28 @@
+﻿
+
+using System.Globalization;
+using System.Text.RegularExpressions;
+using System.Windows.Controls;
+
+namespace Invoice_GenUI.Models.Validation;
+
+public class UsernameValidation : ValidationRule
```

Comment:
:question: in the context of Invoice Generator what is a Username? Looks like you've copy-pasted this code from the web and didn't rename the class. Is this the case?

Reply:
I was watching a video on how ValidationRule works and followed along to the point where they named the class and forgot to rename it once I was done. Inside the class were things that would be useless for the task I was working on.